src/i18n/pt-br/ui.ts

```diff
@@ -103,6 +104,9 @@ export default UIDictionary({
 	'feedback.categoryGroupLabel': 'Escolha a categoria do feedback',
 	'feedback.issue': 'Problema',
 	'feedback.createIssue': 'Criar um problema no GitHub',
```

Comment:
Não faz parte desse PR mas aqui deveria ser `Issue` ao invés de `problema`
```suggestion
	'feedback.createIssue': 'Criar uma Issue no GitHub',
```